portfolio/src/main/webapp/pagination.js

```diff
@@ -0,0 +1,158 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+document.addEventListener('DOMContentLoaded', () => {
+  const commentButton = new CommentButton();
+  commentButton.requestAndDisplayNewCommentsPage();
+});
+
+class CommentButton {
+  constructor() {
+    this.currentCommentsPageIndex = 0;
+    this.totalNumberOfPagesRequested = 1;
+    this.lastPage = -1;
+    this.cursorString = null;
+     
+    document.getElementById('prev-comments-button').classList.add("hidden");
```

Comment:
can you store this element (and others that you reuse) as class variables?


this.prevCommentsButton = document.getElementById('prev-comments-button');
this.prevCommentsButton.classList.add("hidden");  
// etc